config/routes.rb

```diff
@@ -85,6 +85,7 @@
 
   # Doc sections that don't have overview/index pages, so need redirecting
   get "/docs/tutorials",    to: redirect("/docs/tutorials/getting-started"), status: 302
+  get "/docs/governance",   to: redirect("/docs/governance/overview"),       status: 302
```

Comment:
I seem to be breaking from a pattern of _not_ having a redirect like this, but it seems like `/docs/governance` should be more than a 404?